actions/ma-fiater-for-grh-msa.vbs

```diff
@@ -747,6 +747,7 @@ Do
 Loop until are_we_passworded_out = FALSE														' }
 
 Call back_to_SELF
+Dim client_array
 
 DO
```

Comment:
Although you didn't change this, I kept running into a bug where if more than one person is selected, you get a msgbox about selecting one person only. When HH_member_custom_dialog(HH_member_array) is called again, it then adds the HH Membs to the dialog again (so if you have 2 HH members in the dialog to start, you will have 4 after the first msgbox, and then 6 again after it calls HH_member_custom_dialog(HH_member_array) again.